catalog/rocks/entry_catalog.go

```diff
@@ -103,12 +104,14 @@ func NewEntryCatalog(cfg *config.Config, db db.Database) (*EntryCatalog, error)
 		return nil, fmt.Errorf("create tiered FS for committed meta-range: %w", err)
 	}
 
+	pebbleSSTableCache := pebble.NewCache(tierFSParams.PebbleSSTableCacheSizeBytes)
 	metaRangeCache := sstable.NewCache(*cfg.GetCommittedRangeSSTableCacheParams(),
 		metaRangeFS,
-		pebble_sst.ReaderOptions{})
+		pebble_sst.ReaderOptions{Cache: pebbleSSTableCache})
+	pebbleSSTableCache.Ref() // rangeCache keeps a second reference.
```

Comment:
Don't think we should call Ref() here.
SST cache is created here ref=1.
Each service that will accept this cache should call Ref if it keeps the reference (and Unref if it releases it)
In this case when we pass the cache to metaRangeCache and rangeCache each one should call Ref and EntryCatalog should call Unref.

Reply:
Good idea.  Done!